site-src/concepts/api-overview.md

```diff
@@ -171,6 +171,28 @@ The following is required for a Route to be attached to a Gateway:
 1. The Route needs an entry in its `parentRefs` field referencing the Gateway.
 2. At least one listener on the Gateway needs to allow this attachment.
 
+#### Referencing Gateways
+
+A Route can reference a Gateway by specifying the namespace(optional if the
+Route and the Gateway are in the same namespace) and name of the Gateway in
+a `parentRef`. A Route can further select a subset of listeners under the
+Gateway using the following fields in `parentRef`:
+
+1. **sectionName** When `sectionName` is set, the Route selects the listener
+   with the specified name.
+2. **port** When `port` is set, the Route selects all listeners listening on
+   the specified port and with protocol compatible with this kind of Route.
+   It’s not recommended to set `port` unless the networking behaviors specified
+   in a Route must apply to a specific port as opposed to a listener(s) whose
+   port(s) may be changed.
```

Comment:
I personally don't think we need this level of detail here. I'd recommend removing it, but fine to leave it in if anyone feels differently.

Reply:
Sounds good. I think users can always refer to the API documentation for more details.